packages/ckeditor5-media-embed/src/mediaembedediting.js

```diff
@@ -208,19 +214,18 @@ export default class MediaEmbedEditing extends Plugin {
 		// Model -> View (url -> data-oembed-url)
 		conversion.for( 'editingDowncast' ).add(
 			modelToViewUrlAttributeConverter( registry, {
+				elementName,
 				renderForEditingView: true
 			} ) );
 
 		// View -> Model (data-oembed-url -> url)
 		conversion.for( 'upcast' )
 			// Upcast semantic media.
 			.elementToElement( {
-				view: {
-					name: 'oembed',
-					attributes: {
-						url: true
-					}
-				},
+				view: element => [ 'oembed', elementName ].includes( element.name ) && element.getAttribute( 'url' ) ?
+					{ name: true } :
+					null,
+
```

Comment:
Could you please remove this empty line?

Reply:
Done in 780250c